src/stable_browser.ts

```
@@ -810,7 +830,66 @@
+      } catch (e) {
+        if (performance.now() - startTime > timeout) {
+          throw e;
+        }
+      }
+      await new Promise((resolve) => setTimeout(resolve, 3000));
+    }
+  }
+  async simpleClickType(elementDescription, value, _params?: Params, options = {}, world = null) {
+    const startTime = Date.now();
+    let timeout = 30000;
+    if (options && options.timeout) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)